### DISCUSSION TITLE
Is it possible to shorten depName in PR description/branch name for regexmanager with full github url?

### DISCUSSION THREAD

**Discussion by HenrikPoulsen** [renovatebot/renovate#18391] [Request Help] [answered]
I have created a regex manager that looks something like this
```
{
      description: 'Some description,
      fileMatch: [
        '.*\\.yml$'
      ],
      datasourceTemplate: 'git-tags',
      matchStrings: [
        '.*someprefix: (?<depName>.*.git)@(?<currentValue>(v)?[.0-9a-zA-Z\-\_]+)',
      ],
    },
```
which would match lines like `- someprefix: git@github.myorg.mycompany.com:myorg/therepo.git@v1.0.0`
This works and creates a PR with the title of:
> Update dependency git@github.myorg.mycompany.com:myorg/therepo.git to v1.0.2

with the branch name `renovate/gitgithub.myorg.mycompany.com-therepo.git-1.x`

The PR title and branch name is not very useful. It's very verbose and it's not easily understandable what it's updating.

So I want to know if there is any way to shorten this for regex managers where the depName has to be a full url?

I tried using `commitMessageTopic` to see if I could use the `replace` handlebar utility to at least get the PR topic to be more clear and concise, but that does not seem to do anything for the regex manager.
Is there any trick I can use to clean it up?

**Comment by viceice** [comment, marked as answer]
match the full name as packageName and the short name as depName

**Reply by HenrikPoulsen** [reply]
Ah very nice. That works thanks
